odl/contrib/fom/test/test_supervised.py

```diff
@@ -68,5 +68,62 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_mean_value_difference_sign():
+    space = odl.uniform_discr(0, 1, 10)
+    I0 = space.one()
+    I1 = -I0.copy()
```

Comment:
`-` creates a copy (where else would the result be stored?), no need to do it twice